man/preupg.h2m

```diff
@@ -1,26 +1,20 @@
 # This file contains information (complement to the help message of the Preupgrade Assistant) that is to be used by help2man tool to generate a man page.
 # The following is a command to generate the man page:
-# help2man preupg --no-info -o preupg.1 -i preupg.h2m
+# help2man bin/preupg --no-info -o man/preupg.1 -i man/preupg.h2m
 
 [NAME]
 Preupgrade Assistant - advises on the feasibility of system upgrade or migration
 
 [SYNOPSIS]
-preupg [-h, --help]
+preupg [[-h|--help] | [--version] | [--cleanup] | [-l|--list-contents-set]]
 
-preupg [--version]
+preupg [-s|--scan MODULE_SET] | [-c|--contents ALL_XCCDF_PATH] [-d|--debug] [-S|--skip-common] [-m|--mode MODE] [--force] [--text] [--dst-arch ARCH] [--old-report-style] [--select-rules RULES] [-v|--verbose]
 
-preupg [-l, --list-contents-set]
+preupg --list-rules [[-s|--scan MODULE_SET] | [-c|--contents ALL_XCCDF_PATH]]
 
-preupg [-d, --debug] [-S, --skip-common] [-s, --scan MODULE_SET] [-m, --mode MODE] [--force] [--text] [--dst-arch ARCH] [-c, --contents ALL_XCCDF_PATH] [--old-report-style] [--select-rules] [-v, --verbose]
+preupg --riskcheck [-v|--verbose]
 
-preupg [--list-rules] [-s, --scan MODULE_SET]
-
-preupg [--riskcheck] [-v, --verbose]
-
-preupg [-u, --upload URL] [-r, --results TARBALL]
-
-preupg [--cleanup]
+preupg -u|--upload URL [-r|--results TARBALL]
 
 [DESCRIPTION]
```

Comment:
Shouldn't there be a description from https://github.com/upgrades-migrations/preupgrade-assistant/pull/337/files?

Reply:
Yes, thank you. I have created an issue for that: https://github.com/upgrades-migrations/preupgrade-assistant/issues/338.